### PR TITLE
Add aria-label for widget's toolbar

### PR DESCRIPTION
**Pull request by msamsel**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Adds aria-label for widget's toolbar.

---

### Additional information

Requires: ckeditor/ckeditor5-ui#500

Required by:
- https://github.com/ckeditor/ckeditor5-table/pull/201
- https://github.com/ckeditor/ckeditor5-image/pull/302
- https://github.com/ckeditor/ckeditor5-media-embed/pull/88

### CODE REVIEW

File: lang/contexts.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"Widget's toolbar": "The label for assistive technologies used in widget's toolbar."
```
Comment (oleq) [inline lang/contexts.json]:
```suggestion
	"Widget toolbar": "The label describing a toolbar attached to a widget used by assistive technologies."
```

File: src/widgettoolbarrepository.js [hunk 2]
```diff
@@ -117,7 +117,9 @@ export default class WidgetToolbarRepository extends Plugin {
 	 */
 	register( toolbarId, { items, getRelatedElement, balloonClassName = 'ck-toolbar-container' } ) {
 		const editor = this.editor;
-		const toolbarView = new ToolbarView();
+		const locale = this.editor.locale;
+		const t = locale.t;
+		const toolbarView = new ToolbarView( locale, { ariaLabel: t( 'Widget\'s toolbar' ) } );
```
Comment (oleq) [inline src/widgettoolbarrepository.js]: 1. As I pointed out in https://github.com/ckeditor/ckeditor5-ui/pull/500, the `ariaLabel` should be a property.
2. `#register` should allow customization of `ariaLabel` (default `t( 'Widget toolbar' )`).
3. Having the above, we need PRs in 
   * ckeditor5-image
   * ckeditor5-table
   * ckeditor5-media-embed

   that will correctly describe widgets introduced by the features ("Image toolbar", "Table toolbar", etc.)

Comment (msamsel) [reply src/widgettoolbarrepository.js]: I've updated the description for PR to link changes in image, table, media-embed.

File: tests/widgettoolbarrepository.js [hunk 3]
```diff
@@ -89,6 +89,37 @@ describe( 'WidgetToolbarRepository', () => {
 				} );
 			}, /^widget-toolbar-duplicated/, editor );
 		} );
+
+		it( 'should create predefined aria-label for toolbar', () => {
```
Comment (oleq) [inline tests/widgettoolbarrepository.js]:
```suggestion
		it( 'should use a pre–defined aria-label for the toolbar', () => {
```

File: tests/widgettoolbarrepository.js [hunk 4]
```diff
@@ -89,6 +89,37 @@ describe( 'WidgetToolbarRepository', () => {
...
+			const toolbarView = widgetToolbarRepository._toolbarDefinitions.get( 'fake' ).view;
+
+			toolbarView.render();
+
+			expect( toolbarView.element.getAttribute( 'aria-label' ) ).to.equal( 'Widget toolbar' );
+
+			toolbarView.destroy();
+		} );
+
+		it( 'should create custom aria-label if is provided', () => {
```
Comment (oleq) [inline tests/widgettoolbarrepository.js]:
```suggestion
		it( 'should use a custom aria-label when provided', () => {
```

**Pull request merged**